pages/agent/v3/help/_env_get.md

```diff
@@ -0,0 +1,68 @@
+<!--
+  _____   ____    _   _  ____ _______   ______ _____ _____ _______
+ |  __ \ / __ \  | \ | |/ __ \__   __| |  ____|  __ \_   _|__   __|
+ | |  | | |  | | |  \| | |  | | | |    | |__  | |  | || |    | |
+ | |  | | |  | | | . ` | |  | | | |    |  __| | |  | || |    | |
+ | |__| | |__| | | |\  | |__| | | |    | |____| |__| || |_   | |
+ |_____/ \____/  |_| \_|\____/  |_|    |______|_____/_____|  |_|
+
+This file is auto-generated by scripts/update-agent-help.sh, please update the
+agent CLI help in https://github.com/buildkite/agent and run the generation
+script.
+
+-->
+
+### Usage
```

Comment:
The heading levels for this don't look right. I'm guessing it's an issue with the script? If it's not easy to change that, perhaps we should change the structure of the page to make it work.

<img width="457" alt="Screenshot 2023-03-21 at 10 17 29 am" src="https://user-images.githubusercontent.com/119824349/226486077-92496228-68b4-471a-8e94-7e0a92561e3f.png">